Rakefile

```diff
@@ -19,18 +19,21 @@ end
 
 namespace :pkg do
   desc 'Generate package source tar.bz2'
-  task :generate_source => [PKGDIR, "#{BUILDDIR}/modules"] do
+  task :generate_source, [:archive_name] => [PKGDIR, "#{BUILDDIR}/modules"] do |task, args|
 
     version = File.read('VERSION').chomp
     raise "can't read VERSION" if version.length == 0
+    
+    archive_name = args[:archive_name] || "katello-installer-#{version}.tar"
 
     Dir.chdir(BUILDDIR) do
-      `tar -cf #{BUILDDIR}/modules.tar --transform=s,^,katello-installer-#{version}/, modules/`
+      `tar -cf #{BUILDDIR}/modules.tar --transform=s,^,#{archive_name}/, modules/`
     end
 
-    `git archive --prefix=katello-installer-#{version}/ HEAD > #{PKGDIR}/katello-installer-#{version}.tar`
-    `tar --concatenate --file=#{PKGDIR}/katello-installer-#{version}.tar #{BUILDDIR}/modules.tar`
-    `bzip2 -9 #{PKGDIR}/katello-installer-#{version}.tar`
+    `git archive --prefix=#{archive_name}/ HEAD > #{PKGDIR}/#{archive_name}`
+    `tar --concatenate --file=#{PKGDIR}/#{archive_name} #{BUILDDIR}/modules.tar`
+    `gzip -9 #{PKGDIR}/#{archive_name}`
+    `mv #{PKGDIR}/#{archive_name}.gz #{PKGDIR}/#{archive_name}`
```

Comment:
https://github.com/Katello/katello-installer/archive/master.tar.gz does

But even if it didn't, we should rename GitHub's not this one, a gzipped tar should end with `.tar.gz`.